options/posix/generic/netdb-stubs.cpp

```diff
@@ -316,9 +316,59 @@ struct protoent *getprotoent(void) {
 	__builtin_unreachable();
 }
 
-struct servent *getservbyname(const char *, const char *) {
-	__ensure(!"Not implemented");
-	__builtin_unreachable();
+struct servent *getservbyname(const char *name, const char *proto) {
+	int iproto = -1;
+	if (proto &&(!strncmp(proto, "tcp", 3) || !strncmp(proto, "tCP", 3)))
```

Comment:
This seems like a typo, is it actually written like that? (tCP)

Reply:
Fixed, thanks.